### ISSUE TITLE
Polymer 3 support

### ISSUE THREAD

**Issue by Ortham** [ellipticaljs/paper-autocomplete#121]
This may be a bit premature, but are there any plans for this element to support Polymer 3, i.e. ship as an ES6 module published on the NPM registry?

**Comment by jhuesos** [comment]
Yeah, it is in the plans, but it might take a while. The other option i am
thinking is a complete re-write to be only Polymer 3 compatible and to only
provide the pop-up part. The input field part should be handled by the
user, but they are just my first thoughts on this. What do you think?

**Comment by Ortham** [comment]
I think that would be more flexible, but I don't have any strong opinion either way. The element works well for me as it is. 🤷‍♂️

**Comment by madiyetov** [comment]
Hi @jhuesos It would be nice to publish v3.0 with the same API and only after that adding new features. Developers already requiring v3.0. Thanks.

**Comment by jhuesos** [comment]
Noted. I'm this week on holidays outside the country. I will do my best to
do this once I'm back. Of course, PR are always welcome :)

**Comment by madiyetov** [comment]
@jhuesos https://github.com/stat92/paper-autocomplete/tree/3.0-preview . Here analysis.json doesn't generated properly and components page doesn't have link to the demo. But component works OK. You can use it.

**Comment by s-oliver** [comment]
I'm also interested in a Polymer 3 version (npm package). 
I agree with your idea to only handle the popup part, that provides more flexibility - you could add autocompletion to any arbitrary elemtent that way.

**Comment by JoernT** [comment]
is this still in the works or stalled? Just asking as we migrate to Polymer 3 and rely on paper-autocomplete (which is fine as is). Eventually we'll do the migration ourselves.

**Comment by ThomasOrlita** [comment]
Someone made a Polymer 3 version, it works okay for me:
https://www.webcomponents.org/element/@fluidnext-polymer/paper-autocomplete/
